# Generated by Django 5.2.3 on 2025-07-22 11:06

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('ics_app', '0020_rdasempedido_erro_rda'),
    ]

    operations = [
        migrations.AddField(
            model_name='dadosiniciativa',
            name='data_emissao_rda',
            field=models.DateField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='dadosiniciativa',
            name='email1_enviado',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='dadosiniciativa',
            name='email2_enviado',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='dadosiniciativa',
            name='erro_rda',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='dadosiniciativa',
            name='progresso',
            field=models.IntegerField(default=0),
        ),
    ]
